### ISSUE TITLE
Shall it provide a serial interface?

### ISSUE THREAD

**Issue by hcs-svn** [chunter1/precipitationSensorESP32#4]
To be able to use it without WiFi

Other than for LGW it perhaps does not make sense here because usually there will be no server to plug it into that is outside where it rains :-)

Perhaps only for "factory reset" and so on, when the settings are junk.

**Comment by chunter1** [comment]
In my oppinion, we should only keep it for development/debug purpose.

**Comment by hcs-svn** [comment]
Yes. The only reason at the moment would be a reset of the settings to be able to configure once again from scratch. But this can also be done with a flash reset:
`esptool.py --port COM3 erase_flash
`
I also think that it is not required at the monent.
